### DISCUSSION TITLE
How can I find the total difference of values ​​with two different boolean types in a table?

### DISCUSSION THREAD

**Discussion by violetbee** [prisma/prisma#16724] [Q&A] [answered]
For example, I have a Likes table with an isLike column in it, where I want to learn the total number of true values ​​minus the total number of false values.

**Comment by violetbee** [comment]
Can you please help me?

**Comment by violetbee** [comment]
I am sorry for flood but is there anyone can help me?

**Comment by nurul3101** [comment, marked as answer]
Hello @violetbee 👋 

Here's an example on how you can find the difference between two boolean values:

Considering this model:
```prisma
model Likes {
  id     Int     @id @default(autoincrement())
  isLike Boolean
}
```

index.ts
```typescript
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

async function main() {
  // Create Likes
  await prisma.likes.createMany({
    data: [
      {
        isLike: true,
      },
      {
        isLike: false,
      },
      {
        isLike: true,
      },
      {
        isLike: true,
      },
    ],
  });

  const trueLikes = await prisma.likes.count({
    where: {
      isLike: true,
    },
  });

  const falseLikes = await prisma.likes.count({
    where: {
      isLike: false,
    },
  });

  const difference = trueLikes - falseLikes;
  console.log({ difference });
}

main()
  .catch((e) => {
    throw e;
  })
  .finally(async () => {
    await prisma.$disconnect();
  });
```
Here's the response:
<img width="505" alt="Screenshot 2022-12-15 at 6 49 19 PM" src="https://user-images.githubusercontent.com/13049676/207869222-b808270d-3241-4d7d-9054-fe7ed76cbe05.png">

**Reply by violetbee** [reply]
Thank you sir, I thought there could be a query that do the same thing in Prisma

**Reply by nurul3101** [reply]
No worries! 😄

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
